student.rb

```diff
@@ -1,15 +1,40 @@
+require_relative 'classroom'
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :name, :age, :parent_permission, :classroom
+
   def initialize(classroom, name = 'unknown', age = nil, parent_permission: true)
     super(name, age, parent_permission: parent_permission)
     @classroom = classroom
+    @name = name
+    @age = age
+    classroom.add_student(self)
   end
```

Comment:
- [ ] It would be nice if you make a method to add students into the classroom to complete the many_to_many relationship :+1: such as below :arrow_down:

```ruby
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end

```